pkg/config/datafileprojectconfig/config.go

```diff
@@ -82,6 +83,38 @@ func (c DatafileProjectConfig) GetAttributeID(key string) string {
 	return c.attributeKeyToIDMap[key]
 }
 
+// GetAttributes returns attributes
+func (c DatafileProjectConfig) GetAttributes() []entities.Attribute {
+//	return c.attributeSlice
+
+	attributes := []entities.Attribute{}
```

Comment:
Looking at this code, since you are using a Map for attributes, you see the duplicate ID's issue you mentioned. I would be curios to know if we are allowed duplicate Attribute ID's or not... But in regards to why we see this issue showing in the Go SDK and not in others, it could be if other follow similar to the Java SDK, all attributes for the config are stored in an Array rather than a Map since being taken from the datafile. So we just loop through the Attributes and create the OptimizelyAttributes form the projectConfig Attributes. That way no ID's are overwritten. Perhaps we can do that here as well?